packages/eslint-plugin-lit-a11y/lib/rules/accessible-emoji.js

```diff
@@ -92,4 +85,4 @@ const AccessibleEmojiRule = {
   },
 };
 
-module.exports = AccessibleEmojiRule;
+module.exports = ruleExtender(AccessibleEmojiRule, HasLitHtmlImportRuleExtension);
```

Comment:
> ok my solution relies on the ImportDeclaration visitor in createAdditionalVisitors running before the subclass rule, I hope that's a good assumption

Does ruleExtender just merge just two objects together? Or does the order matter?

Reply:
The second arg is its own type, with it's own special methods